### DISCUSSION TITLE
Is it possible to log in bridged account in Bluesky?

### DISCUSSION THREAD

**Discussion by ImMALWARE** [snarfed/bridgy-fed#1635] [Q&A] [answered]
I've created bridge from Mastodon to Bluesky and I would like to to set bridged account as Discord's Bluesky integration account in my profile. But to do that, I need to log in. So, is it possible to somehow log in to bridged account just to verify Discord?

**Comment by ImMALWARE** [comment, marked as answer]
Oh, i saw faq already

**Reply by Tamschi** [reply]
To clarify: No, it's not possible or currently planned.
